litmus/preSi.ml

```diff
@@ -2002,20 +2005,77 @@ module Make
         O.o "}" ;
         O.o ""
 
+      let dump_init_pauth_def env test =
+        dump_choose_def env test ;
+        O.o "// read the SCTLR_EL1 status register" ;
```

Comment:
I have two high level remarks
  1. This code is "constant" (_.i.e._ does not change with litmus input of command-line), It would be better (1) for it to be in its own file, for instance `litmus/libdir/_aarch64/kvm_auth.c` and (2) to include the file with `Insert.insert O.o "kvm-auth.c"`.
  2. Inclusion of this code should depend on some variant such as `pac`, in order not to include this code systematically.